pkg/util/clihelper/clihelper.go

```diff
@@ -117,6 +117,10 @@ func HasPodProblem(pod *corev1.Pod) bool {
 }
 
 func CheckHelmVersion(output string) error {
+	if !strings.HasPrefix(output, "v") {
+		output = "v" + output
+	}
+
 	if semver.Compare(output, MinHelmVersion) == -1 {
```

Comment:
Hello, thanks for sending a PR. Could you add some unit tests for this ? Would like to track the types of version that we are supporting.